runtime/doc/options.txt

```diff
@@ -5292,9 +5292,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 	unescaping, so to keep yourself sane use |:let-&| like shown above.
 							*shell-powershell*
 	To use PowerShell: >
-		let &shell = has('win32') ? 'powershell' : 'pwsh'
+		let &shell = executable('pwsh') ? 'pwsh' : 'powershell'
 		let &shellcmdflag = '-NoLogo -NoProfile -ExecutionPolicy RemoteSigned -Command [Console]::InputEncoding=[Console]::OutputEncoding=[System.Text.Encoding]::UTF8;'
-		let &shellredir = '2>&1 | Out-File -Encoding UTF8 %s; exit $LastExitCode'
+		let &shellredir = '-RedirectStandardOutput %s -NoNewWindow -Wait'
 		let &shellpipe = '2>&1 | Out-File -Encoding UTF8 %s; exit $LastExitCode'
```

Comment:
If `>` doesn't work with powershell then why is it used for 'shellpipe'?

Reply:
There are two cases:

1. `Start-Process sort -RedirectStandardInput filein.txt -RedirectStandardOutput fileout.txt`: requires redirection.
2. `Start-Process dir`: does not require redirection

In the second case, if the command `dir` does not exist, that is, if the user provides a wrong command, nvim needs to write the error to a file (at least that's my understanding, from [tests like this](https://github.com/neovim/neovim/blob/b7084fef4c850d0352488b14dcff0f36a7e75e1c/test/functional/ex_cmds/make_spec.lua#L24-L31)). Now, pwsh doesn't let users direct both std in and err to the same file, that is, it doesn't let us use something like `2>&1`. If you give the same file to both `-RedirectStandardOutput` and `RedirectStandardError`, pwsh will give error. Stackoverflow suggested simply using `Out-File` to redirect everything to a file.

But `Out-File` doesn't work with case 1. My understanding is that, this use case is why vim has two different options for redirection and piping. But I don't have any confirmation except that it seemed to work.